### PR TITLE
Rename Microsoft.Testing.Framework and related projects to Microsoft.Testing.Internal.Framework

### PR DESCRIPTION
**Pull request by nohwnd**
Microsoft.Testing.Framework namespace is used by the new mstest engine. To avoid confusion rename "TATF" to Microsoft.Testing **.Internal.** Framework

### CODE REVIEW

File: test/UnitTests/Microsoft.Testing.Platform.UnitTests/Messages/TestNodeUidTests.cs [hunk 1]
```diff
@@ -1,9 +1,11 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using Microsoft.Testing.Framework;
+using Microsoft.Testing.Internal.Framework;
 using Microsoft.Testing.TestInfrastructure;
 
+using FrameworkTestNodeUid = Microsoft.Testing.Internal.Framework.TestNodeUid;
+
```
Comment (Evangelink) [inline test/UnitTests/Microsoft.Testing.Platform.UnitTests/Messages/TestNodeUidTests.cs:8]:
```suggestion
using FrameworkTestNodeUid = Microsoft.Testing.Internal.Framework.TestNodeUid;
```

**Review by Evangelink — COMMENTED** [review]
I would expect that you have to update the source gen tooling before moving this PR forward.

**Comment by nohwnd** [comment]
Should work now, Windows tests passed, but not linux, because I screwed up the name casing in csproj, and linux file system is case sensitive.

**Pull request closed without merging**